src/test/java/emissary/util/shell/ExecutrixTest.java

```diff
@@ -80,6 +83,27 @@ public void testExecutrixParams() {
         assertTrue("names must use in file ending", names[Executrix.IN].endsWith(".in"));
     }
 
+    @Test
+    public void testExecutrixUniqueBase() {
+        this.e.setInFileEnding(".in");
+        this.e.setOutFileEnding(".out");
+        this.e.setOrder("NORMAL");
+
+        final int COUNT = 1000;
+        final ConcurrentHashMap<String, Integer> concurrentBasePathMap = new ConcurrentHashMap<>();
+        final Set<String> concurrentBasePathSet = concurrentBasePathMap.newKeySet();
+
+        // Generate 100 sets of names
+        IntStream.range(0, COUNT).parallel().forEach(number -> {
+            final String[] name = this.e.makeTempFilenames();
+            assertNotNull("name null DIR", name[Executrix.DIR]);
+            assertNotNull("name null BASE", name[Executrix.BASE]);
+            assertNotNull("name null BASE_PATH", name[Executrix.BASE_PATH]);
+            assertTrue(concurrentBasePathSet.add(name[Executrix.BASE_PATH]));
```

Comment:
Remove this assert statement. Will be handled on line 104.

Reply:
Done.